index.ts

```diff
@@ -16,10 +18,10 @@ export function  checkWebPSupport(): Boolean  {
         return (m[1] === 'Firefox' && m[2] >= 65) || (m[1] === 'Edge' && m[2] >= 18)
     }
 
-    m = navigator.userAgent.match(/OS X\s?(?<os>\d+)?.+ Version\/(?<v>\d+\.\d+)/)
+    m = navigator.userAgent.match(/OS X\s?\d*_?(?<os>\d+)?.+ Version\/(?<v>\d+\.\d+)/)
```

Comment:
This change add a false positive on desktop Safari Version 15.2 (15612.3.6.1.8, 15612)

Reply:
I'm sorry, what do you mean? Safari >= 14 supports webp if the system version is >= 11. I guess we should do a mobile version check as well.

Reply:
webp is not supported in the specified Safari version but the library returns `true` after this change:

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/11010444/184350840-d1a25148-f0ea-4e44-8dd9-1b292734cc38.png">

User agent: 
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.2 Safari/605.1.15`

Reply:
Is that some other webp check tool? [Can I use](https://caniuse.com/webp) is positive webp works on Safari 14+
<img width="441" alt="Bildschirmfoto 2022-08-12 um 6 26 08 PM" src="https://user-images.githubusercontent.com/8287493/184353968-f428a7d2-327b-4a0d-be18-edd343fb8959.png">

Reply:
The tool is in this repo: https://github.com/ihordiachenko/supports-webp-sync/tree/master/supports-webp-app

It works in Safari 14+, but only if the OS version is Big Sur or later

Reply:
this is what I see when I run the tool:
<img width="1030" alt="Bildschirmfoto 2022-08-12 um 7 49 03 PM" src="https://user-images.githubusercontent.com/8287493/184367146-9465f0d1-bd42-4482-a762-82bbf73dca43.png">

It looks like it's the same browser & system as yours but your machine for some reason doesn't display the image?

Reply:
What's your OS version? Mine is macOS Catalina 10.15.7

Reply:
here's my machine:
<img width="516" alt="Bildschirmfoto 2022-08-12 um 7 57 19 PM" src="https://user-images.githubusercontent.com/8287493/184368664-eb4554c1-8797-47ca-82c1-ddedf845451a.png">
I did a quick google search and apparently Apple froze Safari user agent to be 10.15.7 so it's no longer reliable... What a bummer. Also, indeed my regex is not applicable in our case

Reply:
I found another feature that only works on Safari when macOS version is 11+
`window.Intl.ListFormat == null`, if true - you're not on Big Sur or newer. Can you please verify if that's the case on your machine?

Reply:
Great catch, works for me now. I think we can neglect Safari v14.0 given the 0.13% market share and merge it as is.